static/js/impala/suggestions.js

```diff
@@ -195,6 +195,13 @@ $.fn.searchSuggestions = function($results, processCallback, searchType) {
         clearCurrentSuggestions(e);
     });
 
+    var $sel = $results.find('.sel');
+    if ($sel.length) {
```

Comment:
Jquery short circuits if a selector doesn't find anything so you don't need to check the length before setting the click. You could also avoid the var `$sel` and just chain everything in that case.